apps/dotcom/client/package.json

```diff
@@ -67,6 +68,7 @@
 		"identity-obj-proxy": "^3.0.0",
 		"json5": "^2.2.3",
 		"lazyrepo": "0.0.0-alpha.27",
+		"regexgen": "^1.3.0",
```

Comment:
this library seems unsupported (last update 7 yrs ago).
have you used it before / any alternatives?

Reply:
no alternatives, but I have used it before - never really had issues with it, but i probably wouldn't put it somewhere it was accepting untrusted strings i guess?